include/matx/core/utils.h

```diff
@@ -45,6 +45,30 @@
 
 namespace matx {
 namespace detail {
+
+#if 0
+__MATX_INLINE__ cuda::experimental::stf::data_place getDataPlace(void *ptr) {
```

Comment:
Why don't we keep it ? Note that for void data interface it's not super critical but still ...